kuibit/gw_utils.py

```diff
@@ -529,3 +529,43 @@ def signal_to_noise_ratio_from_strain(
     return np.sqrt(
         h_fft.inner_product(h_fft, noises=noise, fmin=fmin, fmax=fmax)
     )
+
+def effective_strain(h, *args, window_function=None, **kwargs):
+    r"""Return the effective strain
+
+    If ``window_function`` is not None, the window will be applied to the signal.
+    All the unknown arguments are passed to the window function.
+
+    This is a Kuibit version of h_eff from PostCactus by W. Kastaun.
+
+    :param h: Gravitational-wave strain.
+    :type h: :py:class:`~.TimeSeries`
+    :param window_function: If not None, apply ``window_function`` to the
+                            series before computing the strain.
+    :type window_function: callable, str, or None
+    :param args, kwargs: All the additional parameters are passed to
+                         the window function.
+
+    :returns: Effective Strain
+    :rtype: :py:class:`~.FrequencySeries`
+
+    """
+    from kuibit.frequencyseries import FrequencySeries
```

Comment:
Import statement should be at the beginning of the file, before importing timeseries (to keep them alphabetical)